src/app/backend/code-systems/service.ts

```diff
@@ -0,0 +1,205 @@
+"use server";
```

Comment:
none of this is net-new code. mostly just moving things from the old file into a separate one here